packages/client/src/api/basic/leaderboard.ts

```diff
@@ -0,0 +1,42 @@
+import basicInstance from './basicInstance'
+
+export type Data = {
+  name: string | undefined
+  score: number
+  seconds: number
+  user_id: number | undefined
+}
```

Comment:
Лучше привести к виду:
```suggestion
  export type LeaderboardPostData = {
    score: number
    seconds: number
    user: LeaderboardUserData
  }
  export type LeaderboardUserData ={
    id: number
    login: string
    avatar?: string
  }
```